app/controllers/collaborators_controller.rb

```diff
@@ -58,6 +60,38 @@ def destroy
     end
   end
 
+  # DELETE /collaborators/:id/transfer
+  # id is group's id
+  #
+  # Removes a group of collaborators
+  #
+  def destroy_group
+    group = Group.find(params[:id])
+    resource = params[:resourceable_type].constantize.find(
```

Comment:
We'll need to guard this call to `constantize` on something coming from user input.